3rd/TokenBucket/TokenBucket.h

```diff
@@ -31,7 +31,7 @@ class TokenBucket {
 
     TokenBucket(const uint64_t rate, const uint64_t burstSize) {
         time_ = 0;
-        timePerToken_ = 1000000 / rate;
+        timePerToken_ = 1000 / rate;
```

Comment:
If our `rate` is over `1000` then this will be `0`. Is that really what is desired here?

Reply:
you're right, this is a real problem. with this PR the resolution changed from micro to milli. i'll take a closer look.

Reply:
To be clear, this was always a problem, but we only see it more now because dividing 1000 is much more likely to result in 0 as the result.